src/locales/en.locale.json

```diff
@@ -371,6 +371,7 @@
   "mno_enterprise.templates.dashboard.marketplace.compare.companies_number": "Number of companies using the application",
   "mno_enterprise.templates.dashboard.marketplace.compare.pricing": "Pricing",
   "mno_enterprise.templates.dashboard.marketplace.compare.pricing_not_available": "pricing information not available",
+  "mno_enterprise.templates.dashboard.marketplace.compare.no_review": "Rating not yet available",
```

Comment:
Was this locale missing?

Reply:
Yessir, this one was missing. I assume because the way things worked previously `Nan` was displayed instead of the translated locale.